src/main/resources/config/application-prod.yml

```diff
@@ -15,6 +15,11 @@ faf-api:
     thumbnail-target-directory: ${MOD_THUMBNAIL_TARGET_DIRECTORY}
   replay:
     download-url-format: ${REPLAY_DOWNLOAD_URL_FORMAT}
+  avatar:
+    download-url-base: ${AVATAR_DOWNLOAD_URL_BASE}
+    allowed-file-extensions: ${AVATAR_ALLOWED_FILE_EXTENSIONS}
+    max-size-bytes: ${AVATAR_MAX_SIZE_BYTES}
```

Comment:
Let's define default values for `allowed-file-extensions` and `max-size-bytes`.

Reply:
just in Avatar properties class or here also?